stdlib/atomic.mli

```diff
@@ -55,6 +58,10 @@ val incr : int t -> unit
 (** [decr r] atomically decrements the value of [r] by [1]. *)
 val decr : int t -> unit
 
+(** [is_aligned r] checks cache-alignment of atomic variable. For references
+    allocated with [make], the output may change mid-execution. *)
+val is_aligned : 'a t -> bool
```

Comment:
Why is this needed? 

If this is only needed for the testcase, may I suggest implementing the C function `caml_atomic_is_aligned` in the testcase. There are a number of examples in the testsuite which shows how to implement test cases which involve both OCaml and C code. For example, see `testsuite/tests/callback/test2.ml`.

Reply:
Moved `is_aligned` to tests. Thanks for the example!